.nvmrc

```diff
@@ -1 +1 @@
-v16
+v18.18.1
```

Comment:
I wouldn't use too specific version in nvmrc. maybe just `v18.18`, this is so if there are updates (like node v18.18.2) that fixes bugs.. the developer can update.